lib/git_reflow/sandbox.rb

```diff
@@ -14,10 +14,12 @@ def run(command, options = {})
 
       if options[:with_system] == true
         system(command)
-      elsif options[:loud] == true
-        puts `#{command}`
```

Comment:
I think I see what you did below, just move it into here so we can avoid the extra nested `if`.